server.js

```diff
@@ -25,6 +25,9 @@ var stats = {
     tunnels: 0
 };
 
+// holds the CLI options
+var options;
```

Comment:
Mostly because we already are passing options around so if what we are doing is not enough we should fix that. Alternatively, we could accept the hostname in an ENV var and replace it out before looking for the sub-subdomain instead of using the tldjs module.